manifests/exploded_war.pp

```diff
@@ -37,6 +37,12 @@
     content => template('i2b2/log4j.properties.erb'),
   }
 
+  exec { 'replace-password' :
+    cwd         => "$dir/WEB-INF/conf",
+    command     => 'sed -i \'s#<parameter name="password">axis2</parameter>#<parameter name="password">Z61OaOvXAfsyIJrQ</parameter>#\' axis2.xml',
+    refreshonly => true,
+  }
+
```

Comment:
* How is this resource refreshed?
* What's the point of changing the password when it's available publicly on GitHub?
* What's the point of changing the password in the first place? What can one do by accessing the axis interface? AFAIK, only listing the services.